SmartPlaces.Facilities/lib/IngestionManager.Mapped/src/MappedGraphIngestionProcessor.cs

```diff
@@ -150,7 +156,13 @@ private async Task UpdateOutputSiteAsync(JsonElement siteElement, CancellationTo
                                    IDictionary<string, BasicRelationship> relationships,
                                    JsonElement targetElement)
         {
-            var basicDtId = targetElement.GetProperty("id").ToString();
+            if (!targetElement.TryGetProperty("id", out var basicDtIdProp))
+            {
```

Comment:
Any reason for skipping the log statements on the things below a Site?

Reply:
If there is no id on the Site element itself, something is wrong with the site. I haven't seen a missing site id... yet

Reply:
If the buildingId is empty, there is something wrong with the data. BuildingId is required.

Reply:
From this return, how would the end user know that their data is wrong without intimate knowledge on the required schema? This seems to be silently swallowing the bug.